### DISCUSSION TITLE
Nonexistent tray icon

### DISCUSSION THREAD

**Discussion by stemy2** [audacious-media-player/audacious#906] [Q&A]
Hello,

I enabled the tray icon, but it's not visible. If i close the app or open directly an audio file, the music is (still) playing but is not visible in the notifications zone.

**Comment by radioactiveman** [comment]
Which Audacious version, toolkit (Audacious with Qt or GTK) and desktop environment do you use?

**Reply by stemy2** [reply]
How do i know wich toolkit i use ?

**Reply by jlindgren90** [reply]
Go to File -> Settings. It will say "Audacious is running in Qt Mode" or "in GTK mode".

**Comment by stemy2** [comment]
It happens with both toolkits.
Audacious v. 4.3.1
KDE desktop environment.

**Comment by Neovex** [comment]
Same Problem in GTK classic mode. Slightly different with modern GTK: when started icon seems to appear for a fraction of a second then immediately disappears.
- Mint 21.3 - Cinnamon 6.0.4
- Audacious V4.1 (Ubuntu official package)

**Reply by Neovex** [reply]
Nevermind. Cinnamon has this weird setting that hijacks audio players and hides their icon to remote control them via the taskbars volume applet. If anyone runs into the same issue: right click applet - configure - uncheck "Hide system-trays for compatible players". :anger:

**Reply by SirGeoffreyB** [reply]
Is there a wiki for audacious?
If so, that would be a good tool tip to have in there.
